frozen-abi/Cargo.toml

```diff
@@ -28,6 +27,9 @@ im = { workspace = true, features = ["rayon", "serde"] }
 memmap2 = { workspace = true }
 subtle = { workspace = true }
 
+[dev-dependencies]
+serde_bytes = { workspace = true }
+
```

Comment:
For consistency, I think this can get put with the other dev dependencies under:

```
[target.'cfg(not(target_os = "solana"))'.dev-dependencies]
```